<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Faq_form</class>
 <widget class="QWidget" name="Faq_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Справка</string>
  </property>
  <widget class="QPushButton" name="author_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Автор</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>301</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>280</y>
     <width>311</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>401</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>95</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="operation_button">
      <property name="text">
       <string>Операции</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="search_button">
      <property name="text">
       <string>Поиск</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="day_button">
      <property name="text">
       <string>День</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
